src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/common/layout.yaml

```diff
@@ -0,0 +1,8 @@
+twig_hooks:
```

Comment:
This needs to be fixed as early as possible as incorrect namespaces start to affect another repos https://github.com/Sylius/Stack/issues/16

Issue only mention service definition, but bundle extension should become `sylius_twig_hooks` and `Symfony/Bundle/SyliusTwigHooks` as well.

Reply:
Yes, the main issue is that we'll break the entire Sylius 2.0 alpha if we do that too soon.

Reply:
I would also be in favour of this change, it would be best to prepare the appropriate PRs for Stack and Sylius 2.0, and to merge them directly before the next alpha, so as not to actually make the current alphas stop working, as the whole admin and shop are based on twig hooks.

Reply:
https://github.com/Sylius/Stack/pull/66 (on Sylius stack)